client/stylesheets/_responsive.scss

```diff
@@ -133,12 +207,21 @@
     .homesearchbtn {
     	font-size: 0.9rem;
     }
+    .profileCardLi {
+    	width: 80% !important;
+    }
 }
 
 @media (max-width: 430px) {
     .homeCardLi {
     	width: 60%;
     }
+    .searchResultsCardLi {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `searchResultsCardLi` should be written in lowercase with hyphens